pkg/feature/conditions.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	interval = 2 * time.Second
+	interval = 10 * time.Second
```

Comment:
If this is the timeout to wait on toggling a feature flag in Serving CR then as a side note in our tests we have 30secs.

Reply:
yep,  but not only for serving's ,  servicemesh feature as well. 
Thanks for the info!
i can check what value ossm is using, and update to a better one if needed.

Reply:
i reverted the change for the interval, and will sync from upstream with a logging updates. 
the check will still do on a 2s based, but reducing log flushing